<launch>

  <group ns="jetbot">

      <!-- remap the color_mask_node -->
      <node pkg="aruco_detector" name="aruco_detector_node" type="aruco_detector.py">
          <remap from="in_topic" to="camera/processed"/>
          <remap from="out_topic" to="aruco_tags"/>
      </node>

  </group>

</launch>
